specs/protocol/configurability.md

```diff
@@ -0,0 +1,81 @@
+- **Policy Parameters**: Parameters of the chain that might be changed without
+  breaking consensus. Consensus is enforced by the protocol, while policy parameters
+  may be changeable within constraints imposed by consensus.
+- **Admin Roles**: These roles can upgrade contracts, change role owners,
+  or update protocol parameters. These are typically cold/multisig wallets not
+  used directly in day to day operations.
+- **Service Roles**: These roles are used to manage the day-to-day
+  operations of the chain, and therefore are often hot wallets.
+
+## Consensus Parameters
+
+| Config Property                       | Description                                                                                                                  | Administrator                       |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| [Batch Inbox address](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L176)                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                  | [L1 Proxy Admin](#admin-roles)                      |
+| [Chain ID](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/chainids.json)                              | Unique ID of Chain used for TX signature validation.                                                                         |                                     |
+| [Challenge Period](https://github.com/ethereum-optimism/superchain-registry/pull/44)                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                        | [L1 Proxy Admin](#admin-roles)                      |
+| Data Availability                     | Where L2 transaction calldata is posted.                                                                                     |                                     |
+| Fault Proof Window / Challenge Window | Duration network participants have to challenge the integrity of an output root.                                             |                                     |
+| [Fee margin](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L281-L283)                            | Markup on transactions compared to the raw L1 data cost.                                                                     | [System Config Owner](#admin-roles)                 |
+| Proxy Fee vault contracts             | Proxy contracts which point to implementation contracts that dictate how user fees are distributed.                          |                                     |
```

Comment:
@ben-chain Why are fee vault contracts explicitly called out, but not other L2 predeploys? (this was carried over from your standard config strawman)

Reply:
@mds1 good callout. @blmalone This can be removed in favor of overall predeploys entry, and then we restrict in standard config.